node_follower/src/error.rs

```diff
@@ -0,0 +1,19 @@
+use thiserror::Error;
+
+pub type Result<T = ()> = std::result::Result<T, Error>;
+
+#[derive(Error, Debug)]
+pub enum Error {
+    #[error("uri error")]
+    Uri(#[from] http::uri::InvalidUri),
+    #[error("grpc {}", .0.message())]
+    Grpc(#[from] tonic::Status),
+    #[error("custom error")]
+    Custom(String),
```

Comment:
when would we use this custom error now that we're not calling the follower directly but instead putting it in a sub-crate?

also, do we want to de-dupe these errors from the error enums in the other apps? you've added the `FollowerError` to the error enum in iot verifier and mobile rewards server but they still have the grpc error there as well